Student/Student/Init/FillDatabaseQuery.sql

```diff
@@ -0,0 +1,205 @@
+USE UniversityDatabase
+GO
+
+BEGIN TRY
+	BEGIN TRANSACTION
+
+	DECLARE @CurrentUserId AS UNIQUEIDENTIFIER = NEWID()
+
+	INSERT INTO [dbo].[Room] ([RoomId], [CreatedBy], [CreatedDateTime]) 
+	VALUES 
+		(22, @CurrentUserId, GETDATE()), 
+		(23, @CurrentUserId, GETDATE()), 
+		(24, @CurrentUserId, GETDATE())
+
+	INSERT INTO [dbo].[CivilStatus] ([CivilStatusName], [DisplayName], [CreatedBy], [CreatedDateTime]) 
+	VALUES 
+		('single', 'Single', @CurrentUserId, GETDATE()),
```

Comment:
```suggestion
		('single', 'Single', @CurrentUserId, @datetime),
```
You should use some variable instead of calling GETDATE() continuously.